features/karaf/esh-ext/src/main/feature/feature.xml

```diff
@@ -55,6 +55,17 @@
     <bundle>mvn:org.eclipse.smarthome.binding/org.eclipse.smarthome.binding.yahooweather/${project.version}</bundle>
   </feature>
 
+  <feature name="esh-extensionservice-marketplace" version="${project.version}">
+    <feature>esh-base</feature>
+    <bundle>mvn:org.eclipse.smarthome.extensionservice/org.eclipse.smarthome.extensionservice.marketplace/${project.version}</bundle>
+  </feature>
+
+  <feature name="esh-extensionservice-marketplace-automation" version="${project.version}">
```

Comment:
> We do not have feature validation in place at ESH.

You are wrong!

It has been present from the beginning.

```
commit 8484160501a8eb8fc62f84eef82b44bbf2050877
Date:   Mon Apr 18 10:24:48 2016 +0200
```

Yourself moved it from `features/karaf-verify` to `features/karaf/verify` at the beginning of 2017.

Perhaps you remember that we talked on introduction of the Karaf feature if the verifiction should be enabled at build time to be executed automatically or only be in place to be triggered manually.

It's just about the line `<!--<phase>process-resources</phase>-->` in the POM file, if we bind the goal to a phase or not.

To trigger the verification manually, you can do it as usual using this command:

```
mvn -pl features/karaf/verify org.apache.karaf.tooling:karaf-maven-plugin:verify@verify-esh
```